### ISSUE TITLE
Loss function hyperparameters

### ISSUE THREAD

**Issue by wyh-neophyte** [wuyou3474/AVTrack#9]
Hi, author.
In your released code, I found the constants of L_spar and L_vir are 0.00001 and 48, 
while in the paper the constants are 0.0001 and 50.
Which one is the correct settings?
